modules/apps/object/object-test/src/testFunctional/tests/ObjectSalesforce.testcase

```diff
@@ -104,6 +116,11 @@ definition {
 		task ("And then the user is able to create entries") {
 			ObjectPortlet.addSingleFieldEntryViaUI(entry = "New Entry LPS-162131");
 
```

Comment:
Let's remove this section since the original purpose is the test works smoothly with we not needing care about plans B. If in the future the test start to not pass because this, we can update again.